frontend/src/routes/rooms/[roomId]/RoomHeader.svelte

```diff
@@ -17,15 +35,25 @@
 </script>
 
 <header class="mt-8">
-  Room URL: <span class="whitespace-nowrap">{url}</span>
+  {#if canShare}
```

Comment:
I like that you put this on the left side. I always imagined it on the right, next to 'delete' but now that I see it this way I think it's better because it makes it harder to accidentally delete when the user means to share.